awx/main/models/notifications.py

```diff
@@ -243,8 +243,6 @@ def send_notification_templates(self, status_str):
                 notification_template_type = 'error'
             all_notification_templates = set(notification_templates.get(notification_template_type, []))
             if len(all_notification_templates):
-                # if status_str != 'running':
```

Comment:
Let's go ahead and just delete this commented code.  If we ever need it for some reason, we can always bring it back with source control :)